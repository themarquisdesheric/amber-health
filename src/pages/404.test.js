import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import NotFoundPage from './404';

test('NotFoundPage/404', () => {
  const wrapper = shallow(<NotFoundPage />);

  expect(toJson(wrapper)).toMatchSnapshot();
});
