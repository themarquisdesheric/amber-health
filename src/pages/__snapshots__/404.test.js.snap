// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage/404 1`] = `
<TemplateWrapper>
  <div
    className="four-oh-four flex justify-center items-center text-center px-12"
  >
    <div
      className="pink-modal-wrapper"
    >
      <div
        className="pink-modal-text p-4"
      >
        <h1>
          404
        </h1>
        <p
          className="text-white mt-4"
        >
          This page has not been found... let's 
          <mockConstructor
            className="underline text-black"
            to="/"
          >
            go home
          </mockConstructor>
           instead?
        </p>
      </div>
    </div>
  </div>
  <div
    className="h-12"
  />
</TemplateWrapper>
`;
